File: backend/fix_imports.py
import os

controllers = []
for root, dirs, files in os.walk('src'):
    for file in files:
        if file.endswith('.controller.ts'):
            controllers.append(os.path.join(root, file))

for file in controllers:
    with open(file, 'r', encoding='utf-8') as f:
        content = f.read()
    
    changed = False
    if "from 'auth/permissions.guard'" in content:
        content = content.replace("from 'auth/permissions.guard'", "from './auth/permissions.guard'")
        content = content.replace("from 'auth/permissions.decorator'", "from './auth/permissions.decorator'")
        changed = True
        
    if changed:
        with open(file, 'w', encoding='utf-8') as f:
            f.write(content)
        print(f"Fixed imports in {file}")

File: backend/fixImports.js
const fs = require('fs');
let c = fs.readFileSync('test/phase8.5.certification.ts', 'utf8');
c = c.replace(/from '\.\/src\//g, "from '../src/");
fs.writeFileSync('test/phase8.5.certification.ts', c);
console.log('Fixed imports');
